packages/lesswrong/components/posts/LWPostsItem.tsx

```diff
@@ -46,13 +48,18 @@ export const styles = (theme: ThemeType) => ({
     display: "flex",
     alignItems: "center",
     justifyContent: "space-between",
-    flexDirection: "column",
-    '&::after': {
-      height: 2,
-      content: '""',
-      width: '100%',
-      backdropFilter: theme.palette.filters.bannerAdBlur,
-    },
+    
+    // On the If Anyone Builds It front page, replace the 2px bottom border with
+    // a rectangular element with a blur, so the background can show through
+    ...(isBookUI && theme.themeOptions.name === 'dark' && {
```

Comment:
The first implementation of this had given light-mode post items a double-thickness bottom border, and removed the border entirely from localgroups (and probably some other places).

Reply:
The comment here seems inaccurate in that I don't see it applying only to the frontpage (which I also think it shouldn't). It then I think combines with the other new dark mode style to give things a bottom border in addition, but not sure.

Reply:
Yeah, the question is just whether the cost of blur depends mostly on blurred surface area, or if the blur-element-count also matters. If it turns out blur-element-count matters a lot (eg if a non-Chrome browser can't handle it) then we could preserve the visual effect by having a single large blur element behind the posts list, and separating the post items by transparent margin. But I think it's probably fine as is (without the blur for nested comments).